### ISSUE TITLE
/r/jailbreak say pirated tweaks brick phones.

### ISSUE THREAD

**Issue by steffanjensen** [pwn20wndstuff/Undecimus#2079]
**Describe the feature you'd like**
The jailbreak community keeps saying pirated tweaks bricks phones, that is a lie.

**Describe alternatives you've considered**
Telling the truth.

**Additional context**
Each time i have asked them about pirated tweaks (just for fun), they say it bricks phones, this scares people so they don't trust the jailbreak.

I have attached image. 
[image](https://i.imgur.com/oJvuoo9.png)

**Comment by Hamlock-maneuverr** [comment]
If you are so bold and ignorant to try such a dumb thing.  Try using filza file manager and go ham on your phone.   You don't even need a cracked version. You can totally brick your phone. Don't believe me?  try yourself.  But please do a video while you are doing it. That would be youtube gold.

**Comment by Ben569483** [comment]
It might brick it. Be careful where you are downloading from because some people who pirate tweaks have some malicious code in the tweak sometimes. So I guess, don't get pirated tweaks.

**Comment by steffanjensen** [comment]
> 
> 
> It might brick it. Be careful where you are downloading from because some people who pirate tweaks have some malicious code in the tweak sometimes. So I guess, don't get pirated tweaks.

The same has Windows which is paid software, that is NOT what I am asking.. Somehow you guys always avoid the question. 
Most devs care more about facts than protection including myself. 

The question is SIMPLE: 
Can pirated tweaks brick your phone?

How do you want devs to develop tweaks if the are afraid of bricking their iphone because you somehow have explained that the backup is not storaged on a different chip than the OS is running on. 

This is HUGE !! and somehow you guys say "naah that does not exist, because tweaks can brick your phone"

If "pirated" tweaks can brick your phone, then original tweaks can ALSO brick your phone which is a huge RED FLAG.

So which is it this time ?

Can an app brick your IPhone ?

**Comment by Ben569483** [comment]
If you're gonna be dumb be dumb.
All I'm saying is
MOST LIKELY. THE REAL NON-PIRATED WONT.
Don't expect another reply.
If you wanna risk it, install it. See if it bricks it or not. Right, enough from me. You aren't getting another reply from me.

**Comment by steffanjensen** [comment]
So you are saying it's possible for non-pirated apps also to brick your phone ?

So how do you have this knowledge, can you show me a source or two ?

Bwcause I talked to one of the main guys yesterday and he told me it was not possible to brick a phone.

Tldr: I am asking for proof than non-pirated apps can brick your phone and the guy above me calls me dumb.. lol

**Comment by Hamlock-maneuverr** [comment]
He's saying your being dumb because you are. There is an old saying in the world of programming and coding.  "Never say never".
Yes Pirated AND non-pirated tweaks have the potential to brick your phone. Your chances of getting a virus or something like that, are much greater from a pirated tweak simply because nobody does anything for free.  There is always a catch.  But anyone with root access to your phone can own you if they really wanted to.  Will they try?  Highly unlikely. Can it be done?  Don't be "dumb". Of course it can.  With enough motivation, anything is possible.

**Comment by steffanjensen** [comment]
We are on github and you say nothing is free are you kidding me.. what kind of statement is this, can we get people to answer this who has contribiuted to this project.

**Comment by Hamlock-maneuverr** [comment]
You don't get the answer you want.  So you choose to insult and try to discredit...  
You have a good day sir.  Hope you find what you're looking for.

**Comment by steffanjensen** [comment]
the answer i want is sources not just "talk from you".. 

Somehow the only answer i want is sources on your claims and when i ask for it, you say i insult and discredit you.. lol

Sorry i don't trust a random guy on the internet..

**Comment by steffanjensen** [comment]
@pwn20wndstuff can you answer this, can apps brick your phone ?

**Comment by Hamlock-maneuverr** [comment]
Pwn is not going to respond to you.  You might as well go play fortnight or make a tiktok video about something pointless.  Just like this post.

**Comment by steffanjensen** [comment]
i will wait :)

**Comment by Hamlock-maneuverr** [comment]
While you wait.  Why don't you check and see the last time pwn posted anything here.  Then, check the time before that.

Such a stupid question to ask BTW.

Have fun waiting...

**Comment by Hamlock-maneuverr** [comment]
What makes this funny is how you have taken the perspective that we are calling you dumb. When in fact, you yourself are telling everyone how dumb you are.

**Comment by Ben569483** [comment]
> the answer i want is sources not just "talk from you"..
> 
> Somehow the only answer i want is sources on your claims and when i ask for it, you say i insult and discredit you.. lol
> 
> Sorry i don't trust a random guy on the internet..

Don’t trust a random guy from the internet... then why you asking for help if you don’t trust a random person? Smh

**Comment by steffanjensen** [comment]
still looking for a source.. and you two can you leave this thread not want to talk to you guys..

**Comment by Ben569483** [comment]
I can’t give proof coz I don’t download pirated tweaks. I always watch what I’m installing. And you invited us to leave so, I will happily

**Comment by steffanjensen** [comment]
if it can't be proven it can't be true.. looking for source still not just "words" of text on a issue on a github.

**Comment by Hamlock-maneuverr** [comment]
Keep waiting.

**Comment by steffanjensen** [comment]
case closed talked to dev, phones can not brick by jailbreaking them or installing apps.

**Comment by Hamlock-maneuverr** [comment]
Congrats.  Then you're all set.
